dice.txt

```diff
@@ -1,7 +1,3 @@
-#### This file describes handfuls of dice you need to calculate
-#### Legend <sides><type> eg. d6 fudge dice look like 6:fudge
-#### 20 numbered - 20:numbered
-#### Loaded 10 sided with 6 loaded - 10:loaded:6
```

Comment:
You should be filtering this out when your code reads the file. Assume that you are going to keep getting files like this on an interval. Do you want to have to hand edit each one before you can run it against your program?